src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -81,7 +81,7 @@
     "python-jose": ("jose",),
     "python-pptx": ("pptx",),
     "pyyaml": ("yaml",),
-    "pymongo": ("bson", "gridfs"),
+    "pymongo": ("bson", "gridfs", "requests"),
```

Comment:
This should say "pymongo" I assume

Reply:
lol wtf! My fingers mutst've been on some kind of autoi-pilot.

```suggestion
    "pymongo": ("bson", "gridfs", "pymongo"),
```